### DISCUSSION TITLE
Slovak Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#395] [Language Leaders]
@x15pa3ck15x, @halecivo, @MartinBucko, @TomasTal - is anyone interested in being a language leader for Slovak?

**Comment by TomasTal** [comment]
I doesn't know if i have the needed knowledge. Or how big the requirements are that are needed. If no one else wants it, then i can try it.

**Comment by halecivo** [comment]
The languages are similar, but I'm native Czech speaker, not Slovak. I could still help if there is not enough people, but native speaker will always do better than me. :-)

**Comment by lipcikm** [comment]
I would be up for it! Happy to contribute for the Slovak part

**Comment by rici44** [comment]
Without programming skill I am ready to help with Slovak language.

**Comment by synesthesiam** [comment]
Just merged a from from @LubosKadasi. Tagging them here for this discussion.

**Reply by LubosKadasi** [reply]
Hello Michael & Thank You.

We can continue discussion here / **[SK]** tu môžeme pokračovať v diskusii a návrhoch: https://github.com/home-assistant/intents/discussions/480

**Comment by LubosKadasi** [comment]
I am interested. But due to work & family I will need a backup person.

**Reply by stevoh6** [reply]
I would love to help you :-)

**Reply by stevoh6** [reply]
@LubosKadasi , @synesthesiam what do you think? :)

**Reply by jose1711** [reply]
> I am interested. But due to work & family I will need a backup person.

if the backup person is still sought, i'd be interested

**Reply by LubosKadasi** [reply]
Hello @synesthesiam, may I ask You to add @jose1711 as another language leader. He did a lot of work to make slovak intents to complete coverage again. Thank You.

**Comment by synesthesiam** [comment]
Thanks @LubosKadasi! Added you as a language leader :)

**Reply by rici44** [reply]
Do we have any notes how may I be helpfull?

**Reply by LubosKadasi** [reply]
Hello/Ahoj.
We can continue here/Môžeme pokračovať tu: https://github.com/home-assistant/intents/discussions/480

**Comment by synesthesiam** [comment]
@LubosKadasi and others - would you be able to translate the new responses files into Slovak?

**Reply by LubosKadasi** [reply]
Yes. Maybe today. I saw your pull request. Thank You

**Reply by schizza** [reply]
#731 is being in progress. Will be committed and merged soon.

**Reply by stevoh6** [reply]
@synesthesiam done :)

**Reply by synesthesiam** [reply]
Great, thanks! Looks like there's also this one? https://github.com/home-assistant/intents/pull/757
